importers/importer.py

```diff
@@ -0,0 +1,103 @@
+import os
+import sys
+import psycopg2
+
+CONN = psycopg2.connect(host=os.environ.get('DATABASE_HOST') or '',
+                        dbname=os.environ.get('DATABASE_NAME') or 'energy-dev',
+                        user=os.environ.get('DATABASE_USER') or '',
+                        password=os.environ.get('DATABASE_PASSWORD') or '')
+
+
+def insert_values(query, items):
```

Comment:
I don't think we need this function.

Reply:
I agree. Removed in [e5d6e4d](https://github.com/carleton-cs-energy-analytics/point-analysis/commit/e5d6e4d2255b90651331b883b2896b961494068c).